quickwit-metastore/src/metastore.rs

```diff
@@ -182,7 +182,7 @@ pub trait Metastore: Send + Sync + 'static {
         &self,
         index_id: &str,
         split_state: SplitState,
-        time_range: Option<Range<u64>>,
+        time_range: Option<Range<i64>>,
```

Comment:
Please, I would like to know the reason for this change from u64 to i64.

Reply:
@evanxg852000 you can have some dates which are before 1970 :)

Reply:
@evanxg852000 also a lot of third party do not allow for unsigned 64. 
Postgresql or Java for instance.